File: be_prepared/python/be_prepared.py
sDegreeF = input("Enter temperature in degrees Fahrenheit (F): ")
 
nDegreeF = int(sDegreeF)
 
nDegreeC = (nDegreeF - 32) * 5 / 9
 
print ("Temperature in degrees C is:", nDegreeC)
 
if nDegreeF < 32:
    print ("Pack long underwear!")
 
if nDegreeF > 100:
    print ("Hydrate!!")

File: be_prepared/cpp/be_prepared.cpp
#include <iostream> 

#include "temperature.h"

int main(int argc, char* argv[])
{
    Temperature* lowLimit = new Temperature(32.0, Farenheit);
    Temperature* highLimit = new Temperature(100.0, Farenheit);
    Temperature* tempF;
    Temperature* tempC;
    double input;
    char output[8];

    std::cout << "Enter the temperature in degrees F : ";  
    std::cin >> input;

    tempF = new Temperature(input, Farenheit);
    tempC = new Temperature(tempF, Celsius);
    
    tempC->format(output);
    std::cout << "The temperature in Celsius is: " << output << std::endl;

    if(*tempF > highLimit) {
        std::cout << "Hydrate!!" << std::endl;
    }

    if(*tempF < lowLimit) {
        std::cout << "Pack Long Underwear!!" << std::endl;
    }

    return(0); // exits the program
}